apps/builder/app/builder/features/sidebar-left/panels/pages/address-bar.tsx

```diff
@@ -158,21 +166,26 @@ const CopyPageUrl = ({ pageUrl }: { pageUrl: string }) => {
 export const AddressBar = ({ addressBar }: { addressBar: AddressBarApi }) => {
   const { pathParamNames, pathParams, pageUrl, updatePathParam } = addressBar;
   const id = useId();
+  const { allowDynamicPath } = useStore($userPlanFeatures);
 
   return (
     <Grid gap={1}>
-      {pathParamNames.map((name) => (
-        <Fragment key={name}>
-          <Label htmlFor={`${id}-${name}`}>{name}</Label>
-          <InputField
-            tabIndex={1}
-            id={`${id}-${name}`}
-            value={pathParams[name] ?? ""}
-            onChange={(event) => updatePathParam(name, event.target.value)}
-          />
-        </Fragment>
-      ))}
-      <CopyPageUrl pageUrl={pageUrl} />
+      {allowDynamicPath &&
```

Comment:
I am notnsure we need such granularity as a feature vs just CMS or something? path, code, redirects were supposed to be all part of CMS, no?